### DISCUSSION TITLE
Welcome to Halo Discussions!

### DISCUSSION THREAD

**Discussion by swaaz** [swaaz/Halo#72] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MuhdAdil1** [comment]
The game is asking name after every round, the main problem i faced while playing.
